.github/workflows/build.yml

```
@@ -26,7 +26,11 @@ jobs:
           mkdir build
           cd build 
           cmake .. -DENABLE_TESTS=1 \
-                   -DAUTO_FORMAT=0
+                   -DBUILD_STATIC=1 \
+                   -DAUTO_FORMAT=0 \
+                   -DPCAP_ROOT_DIR="~/home/mason/libpcap" \
+                   -DBOOST_ROOT="~/home/mason/boost" \
+                   -DNCURSES_ROOT_DIR="~/home/mason/ncurses"
```

```suggestion
                   -DPCAP_ROOT_DIR="$HOME/libpcap" \
                   -DBOOST_ROOT="$HOME/boost" \
                   -DNCURSES_ROOT_DIR="$HOME/ncurses"
```

/home/mason/home/mason? That can't be right